SmarTest/Services/AuthService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using SmarTest.DataLayer.Models;
+using SmarTest.Services.Interfaces;
+
+namespace SmarTest.Services
+{
+    public class AuthService : IAuthService
+    {
+        private readonly List<User> _users = new List<User>
+        {
+            new User { Id = 1, FirstName = "student", LastName = "student", IsTeacher = false },
+            new User { Id = 2, FirstName = "teacher", LastName = "teacher", IsTeacher = true },
+        };
+
+        public User LoggedInUser { get; set; } = null!;
```

Comment:
I don't think this is the best place for this, it should either be static / move it in a singleton instance so that we don't lose it between services injections